### DISCUSSION TITLE
Help Connecting HmIP-BWTH Thermostat to HmIP-FALMOT-C12 Actuator via RaspberryMatic

### DISCUSSION THREAD

**Discussion by drjishen** [OpenCCU/OpenCCU#2707] [🇩🇪 Fragen&Antworten] [answered]
Hi everyone,

I'm new to RaspberryMatic and smart home automation. I need help connecting a HmIP-BWTH wall thermostat to a HmIP-FALMOT-C12 floor heating actuator. I have already looked up this topic but could not find a definite solution or guide. I understood that this is possible (at least using HM app) but don't know how to do it in RaspberryMatic.

Could anyone provide a step-by-step guide or point me to resources for:

- Setting up the connection in RaspberryMatic.
- Pairing the HmIP-BWTH with HmIP-FALMOT-C12.
- Configurations needed for optimal operation.

Also, any advice, suggestion or resources would be greatly appreciated.

Thanks a lot!

**Comment by Baxxy13** [comment, marked as answer]
Da hier im deutschen Bereich gefragt wurde...

Eigentlich brauchst du nur "pro Raum / Heizzone" eine Gruppe vom Typ "HmIP-Heizungssteuerung" anlegen und dort das BWTH (des Raumes), den FALMOT-Kanal (des Raumes) und ggf. Fensterkontakte (des Raumes) hinzufügen. Danach kannst du die Einstellungen im virtuellen Heizgruppen-Gerät vornehmen.

Damit sollte das erstmal grundsätzlich laufen.

**Reply by drjishen** [reply]
vielen Dank. 
Entschuldigung, ich habe es in der deutschen Bereich gepostet. Als ich ein bisschen Deutsch kann, hat mir Ihre Antwort geholfen.

**Comment by Bimmerfriend** [comment]
Moin, da ich das gleiche Problem habe, Geselle ich mich mal dazu...

Irgendwie komme ich nicht weiter. Situation wie folgt: Umrüstung FB Heizung von 230V Stellantrieben auf ein Falmot Setup mit entsprechenden Antrieben. Thermostat soll sein BWTH. FALMOT ist angemeldet und installiert. BWTH auch, war ja schon vorher eingebunden. Nun klappt das Thema Gruppe bei mir nicht - ich habe den Kanal des entsprechenden Stellantreibes vom Falmot und den kompletten BWTH in eine Heizungsgruppe gepackt, allerdings wird der Stellantrieb nicht angesprochen / rausgefahren. 

Muss ich noch etwas anpassen oder verändern? Irgendetwas neustarten oder löschen?

Viele Grüße!

**Reply by Bimmerfriend** [reply]
Ich glaube, ich bin weitergekommen... es war die "globale Bediensperre" gesetzt in den BWTH's. Die ist jetzt raus, und ich konnte die Gruppe erstellen und auch die Thermostaten einfügen. Anscheinend muß per kleiner Taste der Anmeldemodus bestätigt werden beim Hinzufügen zu Grupoen?
